File: Algorithms/Implementation/Bill Division.py
# Bill Division

# https://www.hackerrank.com/challenges/bon-appetit/problem

def bonAppetit(bill, k, b):
  # Write your code here
  sum = 0
  for i in range(len(bill)):
    if i != k:
      sum += bill[i]

  a = sum // 2

  print('Bon Appetit' if b == a else b - a)

File: Algorithms/Implementation/Bill Division.js
// Bill Division

// https://www.hackerrank.com/challenges/bon-appetit/problem

// #1
function bonAppetit(bill, k, b) {
  // Write your code here
  let sum = 0

  for (let i = 0; i < bill.length; i++) {
    if (i !== k) {
      sum += bill[i]
    }
  }

  let a = sum / 2

  console.log(a === b ? 'Bon Appetit' : b - a)
}